src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -233,7 +232,7 @@ public Response addGzip(@Context HttpHeaders httpHeaders, InputStream gzip) {
 	public Response add(@Context HttpHeaders httpHeaders, InputStream json) {
 		Span span = createSpan("datapoints_insert", httpHeaders);
 
-		try (Scope scope = tracer.scopeManager().activate(span, false)) {
```

Comment:
This is unrelated to the actual change, right?

Reply:
yep, just a little clean up